### ISSUE TITLE
Improve Accessibility 

### ISSUE THREAD

**Issue by sallar** [buttercup/buttercup#225]
The current UI implementation is not accessible for visually impaired and/or people with other kinds of disabilities.
We are currently using Devtron, which is able to do an accessibility audit (which we are failing currently). It would be beneficial to check that out and fix the problems before releasing version 1.0.

**Comment by renjithgr** [comment]
#275 has been raised to fix https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_html_01
